packages/theme/build/writeDefaultValue.js

```
@@ -0,0 +1,85 @@
+      let subVar = defaultValue.replace(/var\((\S+)\)/g, (subOldVar, subBaseVar) => {
+        let subDefaultValue = mapVar[subBaseVar]
+        if (!subDefaultValue) {
+          return subOldVar
+        }
+
+        return findTrueValue(subDefaultValue)
+      })
+
+      if (subVar.includes('calc(')) {
+        subVar = subVar.replace('calc(', '').replace(')', '').replace('px', '')
+        subVar = eval(subVar) + 'px'
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)